keyboards/cest73/tkm/config.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xBADD
+#define PRODUCT_ID      0xAA55
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    cest73
+#define PRODUCT         "Ten Key Mirrored 2021 Release 1.03"
+
+/* key matrix size */
+#define MATRIX_ROWS 11
+#define MATRIX_COLS 10
+
+/* matrix sz handy ruler:  1   2   3   4   5   6   7   8   9  10  11  12 */
+/* key matrix pins */
+/*    row    handy ruler: r0  r1  r2  r3  r4  r5  r6  r7  r8  r9  r10 */
+#define MATRIX_ROW_PINS { B0, B1, B2, B3, B4, B5, B6, B7, C6, C7, D0 }
+/*    column handy ruler: c0  c1  c2  c3  c4  c5  c6  c7  c8  c9 */
+#define MATRIX_COL_PINS { D1, D2, D3, D4, D5, D6, D7, F0, F1, F4 }
```

Comment:
Apparently you are using the Teensy 2.0 controller board; does the column connected to the `D6` pin work reliably for you? I don't actually have a real Teensy 2.0 for testing, but I have Teensy++ 2.0 (which uses the AT90USB128 chip), and that board also has a LED on the `D6` pin; apparently that LED together with the builtin pull-up resistor result in some intermediate voltage level on the pin, which sometimes reads as 0, producing false keypresses if the pin is used in the matrix.

To fix this problem, you may need to modify the Teensy 2.0 controller board by desoldering the LED, or rework the PCB to avoid using the `D6` pin for the matrix (you may use that pin for other purposes that are compatible with having a LED towards GND on the same pin).

Reply:
So far had no issues, will keep an eye on it.